website/docs/r/persistent_volume.html.markdown

```diff
@@ -29,6 +29,7 @@ resource "kubernetes_persistent_volume" "example" {
 				volume_path = "/absolute/path"
 			}
 		}
+		storage_class_name = "standard"
```

Comment:
This should probably be left out of the example resource, because it should be left empty by default.